packages/fxa-shared/lib/sentry.ts

```diff
@@ -5,6 +5,12 @@
 import * as Sentry from '@sentry/browser';
 import Logger from './logger';
 
+// HACK: allow tests to stub this function from Sentry
```

Comment:
This is a weird but probably least-ugly hack. 

The `Sentry` from `import * as Sentry from '@sentry/browser';` is a convenience object but not actually an export from `@sentry/browser`. 

That means tests can't stub properties of `Sentry` because it's not shared. Both the test module and the subject module have their own `Sentry` instances with references to read-only ES6 exports.